### ISSUE TITLE
INSERT error: SQLITE_CONSTRAINT: UNIQUE constraint failed: sessions.token

### ISSUE THREAD

**Issue by TheRealHotSwap** [3s3s/opentrade#125]
I get an error in the debug on occasion and I'm not sure if its a real issue or if its just telling me the session is being updated.

INSERT INTO sessions VALUES  ('LONG STRING', '1')
INSERT error: SQLITE_CONSTRAINT: UNIQUE constraint failed: sessions.token
UPDATE sessions SET time='1536876401739' WHERE token='LONG STRING'

**Comment by machado-rev** [comment]
Follow the install instructions!

You need to change some values for safety!

**Comment by TheRealHotSwap** [comment]
I did follow the install instructions word for word. Any idea what needs to be changed?

**Comment by machado-rev** [comment]
Did you changed the words on security configuration?

Em sex, 14 de set de 2018 15:40, TheRealHotSwap <notifications@github.com>
escreveu:

**Comment by TheRealHotSwap** [comment]
I assume you are referring to the private_constrants? Yes, I did do that part. Does the first LONG_RANDOM_STRING1 also go somewhere else in the code?

exports.password_private_suffix = 'LONG_RANDOM_STRING1';       <-- Does a random password only go here and nowhere else?


Also the part I posted with "LONG STRING" It actually had a long string in it and not just those words.

Thanks again for your help!
